src/include/daos/policy.h

```diff
@@ -0,0 +1,32 @@
+/**
+ * (C) Copyright 2021-2021 Intel Corporation.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause-Patent
+ */
+/**
+ * This file is part of daos
+ *
+ * include/daos/policy.h
+ */
+
+#ifndef __DAOS_POLICY_H__
+#define __DAOS_POLICY_H__
+#include <daos/common.h>
+
+/** tier placement policy */
+enum tier_policy_t {
+        DAOS_MEDIA_POLICY_IO_SIZE,
```

Comment:
(style)  code indent should use tabs where possible

Reply:
please address.